seg_lapa/config/train.yaml

```diff
@@ -1,5 +1,11 @@
 random_seed: 0  # If None, seeds not set. If int, uses value to seed.
 
+checkpoint_load:
+  resume_training: True
+  resume_wandb_run: True
+  wandb_run_id: "21kgt2w9"
+  checkpoints_filename: "epoch=1-step=9.ckpt"
+
```

Comment:
Hmm, we need a better system. There are 2 different situations:
1. Load a "pre-train" checkpoint and start training from scratch. Example use: model is pre-trained on synthetic data, and now we want to train on real data.
2. Resume a run from where it left off.

The config system needs to reflect them. And needs to be more intuitive than our old system.